crates/autopilot/src/run.rs

```diff
@@ -546,10 +546,14 @@ pub async fn run(args: Arguments) {
         service_maintainer.run_maintenance_on_new_block(eth.current_block().clone()),
     );
 
+    let persistence = Arc::new(
```

Comment:
Why making it `Arc`? Let's just make the `infra::Persistence` clone-able and do a clone and that's it.

Reply:
My bad, I was worried about the s3 client cloning but it also `Arc`ed inside. Removed the redundant `Arc`.